en/components/datachart_series_tooltips.md

```diff
@@ -99,7 +99,7 @@ The following code snippet demonstrates how to add an item tooltip layer to the
         [xAxis]="xAxis"
         [yAxis]="yAxis"
         valueMemberPath="Value"
-        showDefaultTooltip=true>
+        [showDefaultTooltip]="true">
```

Comment:
why are you using [] for showDefaultTooltip?

Reply:
Who are you referring to? There is no need to use brackets, if `showDefaultTooltip` input was using a property, then brackets should be used in order to bind `showDefaultTooltip` to it.